docs/source/azure/examples/resourcegroupsorphanresources-general.rst

```diff
@@ -1,7 +1,7 @@
 .. _azure_orphanresources:
 
-Delete or report on orphan resources (NICs, Disks, Public IPs).
-===============================================================
+Resource Groups - Delete or report on orphan resources (NICs, Disks, Public IPs)
```

Comment:
This one is an Azure Disk, I might just group it with Virtual Machine/Compute since disks only really exist in that context.